plugins/inputs/win_wmi/sample.conf

```diff
@@ -0,0 +1,15 @@
+# Input plugin to query Windows Management Instrumentation
+[[inputs.win_wmi]]
+  # specifies a prefix to attach to the measurement name
+  name_prefix = "win_wmi_"
```

Comment:
Hmmm, this is a global Telegraf option, so I wonder why you use it here **by default**. Please think of the `sample.conf` as the bare minimum configuration to do something useful! Should we maybe remove (or at least comment) this?